helmfile/11-prometheus-operator.yaml

```
@@ -44,6 +44,16 @@ releases:
   - values/kube-prometheus-stack-wc.yaml.gotmpl
 {{ end }}
 
+- name: kube-state-metrics-clusterapi
```

Should this be named in a more generic way?

```suggestion
- name: kube-state-metrics-extra-configmaps
```